src/store/ChartStore.js

```diff
@@ -372,9 +372,10 @@ class ChartStore {
             onSettingsChange,
         } = props;
         this.api = new BinaryAPI(requestAPI, requestSubscribe, requestForget);
-
+        const needsNoTradingTimes = this.mainStore.state.endEpoch
```

Comment:
@nazaninreihani IMO, the chart shouldn't decide to fetch the trading times or not, it just should rely on the passing conditions.

Reply:
you mean we handle it in deriv? for example we make it false when we have startepoch/endepoch?